sources/documents/2_RTB/Verbali/Interni/VerbaleInterno_2024-06-03/capitolo-riunione.tex

```diff
@@ -42,10 +42,10 @@ \subsubsection{Divisione dei ruoli}
 Di conseguenza, il quinto \glossario{sprint} sarà incentrato sul cambio di tecnologie e avrà durata di una settimana e mezza.
 Infine, il team ha stabilito i documenti che dovranno essere aggiornati:
 \begin{itemize}
-	\item \PdP;
-	\item \NdP;
-	\item \AdR\ (aggiornamento dei casi d'uso e inserimento dei grafici);
-	\item \Gls.
+	\item \PdP\ per l'aggiornamento dello \glossario{sprint};
```

Comment:
Queste modifiche sono da rimuovere, come indicato nel commento precedente.

Reply:
Sono derivate dal merge queste modifiche, no? In tal caso dovrebbero venir ignorate in review e merge della PR

Reply:
```
\begin{itemize}
	\item \PdP\ per l'aggiornamento dello \glossario{sprint};
	\item \NdP\ per i motivi spiegati sopra;
	\item \AdR\ per un'aggiornamento sui casi d'uso e l'inserimento dei grafici;
	\item \Gls\ per un aggiornamento dei termini (di minor rilevanza rispetto ai documenti sopra citati).
\end{itemize}
```
ora sono così ho fixato lo spazio che dava problemi in caso in merge ci ricordiamo di controllare

Reply:
```
\begin{itemize}
	\item \PdP;
	\item \NdP;
	\item \AdR\ (aggiornamento dei casi d'uso e inserimento dei grafici);
	\item \Gls.
\end{itemize}
```
Il problema è che avevo fixato in modo diverso, forse potrebbe creare problemi.